src/test/java/seedu/address/logic/commands/ScheduleCommandTest.java

```diff
@@ -1,5 +1,138 @@
 package seedu.address.logic.commands;
 
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_SCHEDULE_DATE_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_SCHEDULE_NAME_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_SCHEDULE_TIME_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.CommandTestUtil.showPersonAtIndex;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Schedule;
+import seedu.address.testutil.PersonBuilder;
+import seedu.address.testutil.ScheduleBuilder;
+import seedu.address.testutil.ScheduleDescriptorBuilder;
+
 public class ScheduleCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void execute_allFieldsSpecifiedUnfilteredList_success() {
```

Comment:
what does unfiltered list mean?

Reply:
it means that the contact list did not apply a filter, so it is accessing all the contacts